_includes/demos/balance-csharp.cs

```diff
@@ -8,53 +8,45 @@
 /*
  * This is an example of using C# to get the balance of an account.
  * The example was originally written in dotnet version 5.0.  
- * it uses the Newtonsoft Json.net Library https://www.newtonsoft.com/json
- * This can be run on Windows/linux or Mac
+ * It uses the Newtonsoft Json.net Library https://www.newtonsoft.com/json
+ * This can be run on Windows/linux or Mac.
  *  
  * To run this example 
  * - Ensure you have dotnet SDK installed https://dotnet.microsoft.com/download/dotnet. 
- * - Run `dotnet new console --output account_balance` In a dir on your computer
+ * - Run `dotnet new console --output account_balance` in a directory on your computer.
  * - Then run `dotnet add account_balance package Newtonsoft.Json` 
- * - Change to the new account_balance dir and edit the `Program.cs` file, paste in the contents of this script and save.
- * - Run `dotnet run --project account_balance ` in the parent dir.  
- *
- * The api token should be from the same account that the balance is to be checked for.
+ * - Change to the new account_balance directory and edit the `Program.cs` file, paste in the contents of this script and save.
+ * - Run `dotnet run --project account_balance` in the parent directory.
+ * The API token should be from the same account that the balance is to be checked for.
  */
 namespace DerivWSDemo
 {
     class DerivWS
     {
         private ClientWebSocket ws = new ClientWebSocket();
-        
-        //You can register for an app_id here https://developers.deriv.com/docs/app-registration/.
-        private string app_id = "1089"; // Change this to yor app_id
-
-        //You can get your token here https://app.deriv.com/account/api-token.
-        public static string token = ""; //Change this to the token you want to get the balance from. 
+        // You can register for an app_id here https://developers.deriv.com/docs/app-registration/.
+        private string app_id = "1089"; // Change this to yor app_id.
+        // You can get your token here https://app.deriv.com/account/api-token.
+        public static string token = ""; // Change this to the token you want to get the balance from. 
         private string websocket_url = "wss://ws.binaryws.com/websockets/v3?app_id=";
 
         public async Task SendRequest(string data)
         {
-
             while (this.ws.State == WebSocketState.Connecting) { };
             if (this.ws.State != WebSocketState.Open)
             {
                 throw new Exception("Connection is not open.");
             }
-
             var reqAsBytes = Encoding.UTF8.GetBytes(data);
             var ticksRequest = new ArraySegment&lt;byte&gt;(reqAsBytes);
```

Comment:
Shouldn't this be ArraySegment<byte> ? Looks like it's using generics or is it something else that I don't know? 🤔

Reply:
Explained this in next comment